lib/app.rb

```diff
@@ -16,6 +16,8 @@ def initialize
     @authors = File.file?('data/authors.json') ? load('data/authors.json')['authors'] : []
     @games = File.file?('data/games.json') ? load('data/games.json')['games'] : []
     @books = File.file?('data/books.json') ? load('data/books.json')['books'] : []
+    @musicalbums = []
+    @genres = []
```

Comment:
What happens if there is something saved in `@musicalbums`? How you retrieve the data?